www/source/get-started/cloudfoundry/index.html.slim

```diff
@@ -20,20 +20,32 @@ div.get-started--container
 
   section.get-started--boxes
     div.row
-      div.columns.small-12.large-8.np-left
+      div.columns.small-12.large-6.np-left
         a.get-started--boxes--item.card.feature-card href="/blog/2018/03/habitat-cloudfoundry"
           h4.h4 Exporting Docker Images for Cloud Foundry with Habitat
+          h6.h6 WALKTHROUGH
+          p
+            | Learn how to create and export a Docker image for Cloud Foundry with this Habitat walkthrough!
+      div.columns.small-12.large-6.np-right
+        a.get-started--boxes--item.card href="/docs/developing-packages/#exporting-to-cloud-foundry"
+          h4.h4 Habitat Documentation for Cloud Foundry
           h6.h6 DOCS
           p
-            | Learn how to create and export a Docker image for Cloud Foundry with Habitat!
-      div.columns.small-12.large-4
+            | Current Habitat documentation for working with and deploying to Cloud Foundry
+      div.columns.small-12
         .get-started--boxes--item
-          h4.h4 Related links
+          h4.h4 Related external links
           h6.h6 Reference
           ul.link-list
             li.link-list--item
-              = link_to 'Exporting to Cloud Foundry', '/docs/developing-packages/#exporting-to-cloud-foundry'
+              = link_to 'Habitat to Cloud Foundry', 'https://www.starkandwayne.com/blog/habitat-to-cloud-foundry/'
+            li.link-list--item
+              = link_to 'Building a scalable and reliable industrial cloud service - Amulya Sharma, GE Digital', 'https://www.youtube.com/watch?v=zaKl59hyWys'
+            li.link-list--item
+              = link_to 'Chef Habitat Addresses the Complexities of Moving Containers into Production', 'https://thenewstack.io/chef-habitat-addresses-issues-moving-containers-production/'
+            li.link-list--item
+              = link_to 'Chef cooks up a new Habitat for packaging apps', 'https://www.cbronline.com/enterprise-it/chef-cooks-new-habitat-packaging-apps/'
             li.link-list--item
-              = link_to 'Cloud Foundry Docs', 'https://www.cloudfoundry.org/docs-download/'
+              = link_to 'Using Habitat to Build the Industrial Cloud at GE Digital', 'https://blog.chef.io/2017/10/12/using-habitat-to-build-the-industrial-cloud-at-ge-digital/'
             li.link-list--item
-              = link_to 'Habitat Docs', '/docs/'            
\ No newline at end of file
+              = link_to 'How Habitat can Boost your Chef Ecosystem', 'https://www.youtube.com/watch?v=F1L3RlNnJqI'
```

Comment:
Title-case would make this `How Habitat Can Boost your Chef Ecosystem `

Reply:
Or `How Habitat can boost your Chef ecosystem` would work, too, but the YouTube page just has it wrong. :smile: